server/conf/application.dev.conf

```diff
@@ -24,5 +24,6 @@ play.filters {
 
 # Feature flags.
 application_status_tracking_enabled = true
+allow_civiform_admin_access_programs = true
```

Comment:
I'd vote for leaving this off by default, since that's the value in the base configuration.

Reply:
Yes, I didn't mean to submit this, just used it for local testing.